selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/client/TestSshServiceClient.java

```diff
@@ -53,7 +69,20 @@ public void deleteMachineKeyByName(String name) throws Exception {
         .request();
   }
 
-  public String generateGithubKey() throws Exception {
+  public synchronized void updateGithubKey() throws Exception {
```

Comment:
It is better to move this method to TestGitHubServiceClient

Reply:
**TestGitHubServiceClient** class is going to be replaced by original GitHub client in near future.